saleor/settings.py

```diff
@@ -658,3 +658,7 @@ def get_host():
     "RELAY_CONNECTION_ENFORCE_FIRST_OR_LAST": True,
     "RELAY_CONNECTION_MAX_LIMIT": 100,
 }
+
+EXTENSIONS_MANAGER = "saleor.core.extensions.manager.BaseManager"
+
+PLUGINS = []
```

Comment:
Note for the documentation: we should tell the users how to set `DJANGO_SETTINGS` to override `PLUGINS` without touching `settings.py`